sfge/build.gradle.kts

```diff
@@ -78,20 +81,27 @@ tasks.test {
 		// Show extra expected info when there is a failure
 		exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
 	}
-	// Run tests in multiple threads
-	maxParallelForks = Runtime.getRuntime().availableProcessors()/2 + 1
+	maxParallelForks = Runtime.getRuntime().availableProcessors()/2 + 1 // Run tests in multiple threads
```

Comment:
OK. I certainly can do that. Might take a little bit to break this habit though.
I tend to do inline comment for single lines of codes and leading comments for blocks of code. But if I use a leading comment for a single line of code, I would follow up that line with a new line so that we know the comment only applies to that one line and not all of the following lines.